setup.py

```diff
@@ -47,25 +29,11 @@ def run(self):
     for item in os.listdir(os.path.dirname(os.path.abspath(__file__)))
     if (os.path.isfile(item) and item.startswith("Dockerfile-"))
 ]
-setup_files = ["setup.py", ".dockerignore", "pyproject.toml", "MANIFEST.in"]
-all_extra_files = dockerfiles + setup_files
-all_extra_files_pathcorrected = [os.path.join("../", item) for item in all_extra_files]
```

Comment:
After installement, user receives this files directly in `site-packages/` folder. The reason is using `../` in the path; however, that is caused by the fact these files are located in repo root.

Good practice for building a wheel by `setup.py` (and, actually, the only possible one) is **not to add** these files to the wheel. Only files that are required for module should be added. Moreover, any file that should be added to user's `site-packages/GANDLF` module should be located in module source library also (`./GANDLF/` in our case, not in a repo root).

So, there are two options:
(1) move necessary files to `./GANDLF/` module source folder;
(2) just don't include them to the wheel at all.

I chose the second solution.